kubetest2/pkg/app/testers/standard/ginkgo/kubectl/kubectl.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubectl
+
+import (
+	"fmt"
+	"os"
+
+	"k8s.io/test-infra/kubetest2/pkg/exec"
+)
+
+const (
+	kubectl = "./kubernetes/cluster/kubectl.sh"
+)
+
+// MasterURL obtains the URL of the k8s master from kubectl
```

Comment:
ServerURL (or APIServerURL) or similar. It's more precise, and this term is legacy.

Reply:
Done.